packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -272,17 +273,42 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
     ): Promise<void> {
         // If GC is disabled, skip setting referenced used routes since we are not tracking GC state.
         if (!this.gcDisabled) {
-            const gcDetailsBlob = snapshotTree.blobs[gcBlobKey];
-            if (gcDetailsBlob !== undefined) {
-                const gcDetails = await readAndParseBlob<IGarbageCollectionDetailsBase>(gcDetailsBlob);
+            let gcDetails: IGarbageCollectionDetailsBase = { gcData: { gcNodes: {} } };
+            const gcSnapshotTree = snapshotTree.trees[gcTreeKey];
```

Comment:
Why did this change from `snapshotTree.blobs["gc"]` to `snapshotTree.tree["gc"]`?

Reply:
Shouldn't https://github.com/microsoft/FluidFramework/pull/13328/files this test start failing?

Reply:
Yes, you are right. I had missed couple of things. I fixed them and the test too.

Reply:
Note: `snapshotTree.trees["gc"]` is specifically for the root `SummarizerNodeWithGC`